### PR TITLE
Rebuild RHEL facts on Vagrant

### LINKED ISSUE #276: IP addresses are different on RedHat and other distributions
I ran into this issue when testing https://github.com/voxpupuli/puppet-openvpn/

The local IP address on Ubuntu systems is `10.0.2.15` while on the RedHat systems it is `10.109.1.2`. The OpenVPN module checks for the IP address in its tests, so the tests need to take into account what the facter says the IP address is. This seems to be unnecessarily complex.

### PR DESCRIPTION
**Pull request by yakatz**
Fixes #276

### CODE REVIEW

**Review by smortex — APPROVED** [review]

File: facts/3.14/redhat-7-x86_64.facts [hunk 1]
```diff
@@ -72,58 +72,71 @@
     "uid": 0,
     "user": "root"
   },
-  "interfaces": "eth0,eth1,lo",
+  "interfaces": "eth0,lo",
   "ipaddress": "10.0.2.15",
-  "ipaddress6": "fe80::5054:ff:fe8a:fee6",
```
[the remark this answers is not in the document]
Comment (yakatz) [reply facts/3.14/redhat-7-x86_64.facts:77]: I ran Vagrant to generate this. Do you know if there is a way to make sure Vagrant has IPv6?

[the remark this answers is not in the document]
Comment (yakatz) [reply facts/3.14/redhat-7-x86_64.facts:77]: Looks like a few more to add

File: facts/3.14/redhat-7-x86_64.facts [hunk 2]
```diff
@@ -210,30 +297,39 @@
...
     }
   },
   "mtu_eth0": 1500,
-  "mtu_eth1": 1500,
   "mtu_lo": 65536,
   "netmask": "255.255.255.0",
   "netmask6": "ffff:ffff:ffff:ffff::",
   "netmask6_eth0": "ffff:ffff:ffff:ffff::",
-  "netmask6_eth1": "ffff:ffff:ffff:ffff::",
-  "netmask6_lo": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff",
```
Comment (bastelfreak) [inline facts/3.14/redhat-7-x86_64.facts:226]: this needs to stay as well

File: facts/3.14/redhat-7-x86_64.facts [hunk 3]
```diff
@@ -210,30 +297,39 @@
...
   "netmask_eth0": "255.255.255.0",
-  "netmask_eth1": "255.255.255.0",
   "netmask_lo": "255.0.0.0",
   "network": "10.0.2.0",
   "network6": "fe80::",
   "network6_eth0": "fe80::",
-  "network6_eth1": "fe80::",
-  "network6_lo": "::1",
```
Comment (bastelfreak) [inline facts/3.14/redhat-7-x86_64.facts:234]: same here

Comment (bastelfreak) [reply facts/3.14/redhat-7-x86_64.facts:234]: I don't see any facts that were missing previously and that you added, so maybe don't edit this factset at all?

File: facts/3.14/redhat-7-x86_64.facts (hunk 2 above)
Comment (yakatz) [reply facts/3.14/redhat-7-x86_64.facts:226]: Should there be multiple network interfaces? I can enable IPv6 for `lo`, but there is no `eth1`

**Comment by zilchms** [comment]
@yakatz how is the status on this? :) stumbled upon errors in a different module due to the inconsistencies you address. any plans on addressing the feedback bastelfreak gave you and getting this PR merged? :)

**Comment by yakatz** [comment]
> any plans on addressing the feedback bastelfreak gave you and getting this PR merged?

If course I have plans :-)
On the other hand, I have a deadline coming up to get my PhD dissertation done and I shouldn't be working on anything but that right now

**Comment by zilchms** [comment]
Dissertation definitely has priority :) good luck on that

**Comment by yakatz** [comment]
Rather than try to rebase this, I am closing this and starting again.

**Pull request closed without merging**